objgraph.py

```diff
@@ -920,6 +923,8 @@ def _short_repr(obj):
             return name + ' (bound)'
         else:
             return name
+    if _isinstance(obj, types.LambdaType):
+        return 'lambda: %s:%s' % (os.path.basename(obj.func_code.co_filename), obj.func_code.co_firstlineno)
```

Comment:
This is nice. 

What is missing is a unit test.  Currently objgraph has 100% test coverage, and I want to maintain that.
There are a bunch of copy-able examples called `test_short_repr_...` in `tests.py`.  I suggest using assertRegex so you don't have to hardcode the line number of the lambda.